src/server/controllers/userAuth.js

```diff
@@ -0,0 +1,30 @@
+const setUserAndSession = (req, res) => {
+  req.session.token = req.user.token;
+
+  // Set user cookies
+  res.cookie(
+    'user',
+    JSON.stringify({
+      name: req.user.displayName,
+      id: req.user.id,
+      avatar: req.user.photos[0].value
+    })
+  );
+
+  res.redirect('http://localhost:3000/');
```

Comment:
please check if just redirect to current url will be enough. In this case if someone changes port or we run project on some server it will fail

Reply:
Need to replace by a constant , because redirecting to   `res.redirect('/');` guides to `localhost:8080/` which is backend root, and that's not what we wanted